browser/extensions/api/brave_sync_event_router.cc

```diff
@@ -108,6 +108,19 @@ void BraveSyncEventRouter::SendGetBookmarksBaseOrder(
   event_router_->BroadcastEvent(std::move(event));
 }
 
+void BraveSyncEventRouter::SendCompactSyncCategory(
+    const std::string& category_name) {
+  std::unique_ptr<base::ListValue> args(
+     extensions::api::brave_sync::OnSendCompactSyncCategory::Create(
+        category_name).release());
+  std::unique_ptr<Event> event(
+     new Event(extensions::events::FOR_TEST,
```

Comment:
it is just a `events::HistogramValue` and I can't find a suitable value from the existing list. Should we extend the enum? Please note that brave sync will eventually go away and migrated to native implementation